lib/parse-message.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.nickCommand = function (message, socket) {
+  socket.username = message.split(' ').slice(1).join(' ').trim();
+  socket.write(`you are now ${socket.username}\n`);
+};
+
+exports.dmCommand = function dmCommand(message, array){
+  // "/dm user_30 hello how are you"
```

Comment:
In the future, be more mindful of taking out unclear comments in your code. I'm telling everyone this to just get in the habit of best practices of pushing comment/console.log-free code before a hypothetical deployment.